ui/lib/pki/addon/components/pki-role-form.hbs

```diff
@@ -62,39 +57,45 @@
               data-test-toggle-group={{group}}
             />
             {{#if (get @model prop)}}
-              <div class="box is-marginless">
-                {{#let (get @model.fieldGroupsInfo group) as |groupInfo|}}
-                  {{! Header }}
-                  {{#if groupInfo.header}}
+              <div class="box is-tall is-marginless" data-test-toggle-div={{group}}>
+                {{#let (get @model.fieldGroupsInfo group) as |toggleGroup|}}
```

Comment:
`toggleGroup` came to mind here, but open to suggestions! Also happy to keep as `groupInfo`